CMakeLists.txt

```diff
@@ -697,8 +698,8 @@ install(TARGETS osrm_store DESTINATION lib)
 # Setup exporting variables for pkgconfig and subproject
 #
 
-if(BUILD_DEBIAN_PACKAGE)
-  include(CPackDebianConfig)
+if(BUILD_PACKAGE)
```

Comment:
Would it make sense to add `option(BUILD_PACKAGE "Build OSRM package" OFF)`  on line 20?